src/client/containers/UserList/onStartChat.js

```diff
@@ -0,0 +1,36 @@
+import { getExistchannel } from '../../api/getExistchannel';
+
+const requestConfig = (body) => ({
```

Comment:
You can actually remove this config and switch to `fetchWithAuth` instead. If you need help, let me know.

Reply:
great idea :)